src/contentscript/multitable-panel/components/image.tsx

```diff
@@ -11,10 +11,15 @@ export interface Props {
 }
 
 export const Image: FC<Props> = ({ image, alt, fallbackUrl }) => {
-  const [imageUrl, setImageUrl] = useState(
-    (image?.ipfs_cid ? `https://ipfs.near.social/ipfs/${image.ipfs_cid}` : image?.url) ||
-      fallbackUrl
-  )
+  const [imageUrl, setImageUrl] = useState<string | undefined>(undefined)
+
+  useEffect(() => {
+    image?.ipfs_cid
+      ? setImageUrl(`https://ipfs.near.social/ipfs/${image.ipfs_cid}`)
+      : image?.url
+      ? setImageUrl(image?.url)
+      : setImageUrl(fallbackUrl)
+  }, [image, imageUrl])
```

Comment:
Why did you rewrite it? Looks like it works the same way, but uses excessive `useEffect`.